src/entity/user/User.ts

```diff
@@ -78,18 +78,15 @@ export class User {
   bio?: string;
 
   @Column({ length: 15, nullable: true })
-  @Index('users_twitter_unique', { unique: true })
```

Comment:
Would this not allow adding duplicate twitter handles? We even check this error in `updateProfile` mutation?

Reply:
Can revert the unique ones to be safe rather.

Reply:
I'd go over unique once and double check which ones we need.